kubernetes/utils/deserialize.py

```diff
@@ -171,8 +171,8 @@ def response_type_from_dict(data, verbose=False):
         elif hasattr(k8s_api, "read_{0}_with_http_info".format(kind)):
             fnc_lookup = "read_{0}_with_http_info".format(kind)
         # Try with the get if no read fnc found
-        # elif hasattr(k8s_api, "get_{0}_with_http_info".format(kind)):
-        #     fnc_lookup = "get_{0}_with_http_info".format(kind)
+        elif hasattr(k8s_api, "get_{0}_with_http_info".format(kind)):
+            fnc_lookup = "get_{0}_with_http_info".format(kind)
```

Comment:
I remember encountering this before but can't replicate it now. Can't seem to find the object I encountered this with (I'm coding new system after switching jobs). I'm still unsure if we should just remove it.

Reply:
`get_{0}_with_http_info` covers the following APIs
```
get_api_group
get_api_resources
get_api_versions
get_code
get_cluster_custom_object*
```

They all have models well-defined, except custom objects which are dynamically defined by user. This line looks good to me and I think we should keep it

Reply:
Sounds good; thanks for your notes.